web/themes/custom/hoeringsportal/templates/content/node--project-main-page--teaser.html.twig

```diff
@@ -0,0 +1,19 @@
+
+<a href="{{ url }}" class="activity-teaser-inner activity-teaser-link-wrap text-decoration-none shadow-sm">
+  <div class="activity-teaser-image">
+    {{ content.field_project_image }}
+  </div>
+  <div{{ attributes.addClass('activity-teaser-content') }}>
+    <h3 class="activity-teaser-title">{{ label[0]['#context'].value|length > 40 ? label[0]['#context'].value|slice(0, 40) ~ '...' : label }}</h3>
```

Comment:
Wouldn't something along the line (!) of https://css-tricks.com/almanac/properties/l/line-clamp/#aa-hey-cant-i-do-this-with-text-overflow make more sense here?

Reply:
Lol - only focused on the description since it did not show.
Used my newly created line-clamp utility class to cut of lines instad.